### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by rgaiacs**

### CODE REVIEW

**Comment by rgaiacs** [comment]
@gvwilson Can you review this?

File: index.html [hunk 1]
```diff
@@ -244,6 +244,45 @@
...
 <hr/>
 
+<h2 id="faq">FAQ</h2>
+<ol>
+  <li>
+    <p><strong>I don't have a international credit card. What I can do?</strong></p>
+    <p>Please contact
+      {% if page.contact %}
+        <a href='mailto:{{page.contact}}'>{{page.contact}}</a>
+      {% else %}
```
Comment (gvwilson) [inline index.html:254]: Please take out the site.contact - we won't know what to do with queries.

File: index.html [hunk 2]
```diff
@@ -244,6 +244,45 @@
...
+        <a href='mailto:{{site.contact}}'>{{site.contact}}</a>
+      {% endif %}
+      as soon as possible. We will handle case by case.
+    </p>
+  </li>
+  <li>
+    <p><strong>Who will receive the money from the entry fee?</strong></p>
```
Comment (gvwilson) [inline index.html:261]: We usually call it a registration fee, not an entry fee.

File: index.html [hunk 3]
```diff
@@ -244,6 +244,45 @@
...
+    <p>The entry fee will directly to Software Carpentry as
```
Comment (gvwilson) [inline index.html:262]: will _go_ directly

File: index.html [hunk 4]
```diff
@@ -244,6 +244,45 @@
...
+    <strong>administrative fee</strong> for the workshop,
```
Comment (gvwilson) [inline index.html:263]: _part of the_ _administrative fee_ for the workshop,

File: index.html [hunk 5]
```diff
@@ -244,6 +244,45 @@
...
+    see <a href="http://software-carpentry.org/scf/workshops.html">this</a> for
```
Comment (gvwilson) [inline index.html:264]: this _page_

File: index.html [hunk 6]
```diff
@@ -244,6 +244,45 @@
...
+    more information.</p>
+  </li>
+  <li>
+    <p><strong>The date is correctly? June 04-05 is Corpus Christi holiday.</strong></p>
```
Comment (gvwilson) [inline index.html:268]: _Is the date correct?_ rather than _The date is correctly?_

File: index.html [hunk 7]
```diff
@@ -244,6 +244,45 @@
...
+    <p><strong>Yes</strong>, the date is correctly.</p>
```
Comment (gvwilson) [inline index.html:269]: _correct_ rather than _correctly_

File: index.html [hunk 8]
```diff
@@ -244,6 +244,45 @@
...
+  </li>
+  <li>
+    <p><strong>Organizers will provide free lunch?</strong></p>
```
Comment (gvwilson) [inline index.html:272]: _Will the organizers provide a free lunch?_

File: index.html [hunk 9]
```diff
@@ -244,6 +244,45 @@
...
+    <p><strong>No.</strong> We are doing our best to make the workshop affordable
+    to everyone and because of it lunch isn't included.
+    You can check <a href="#restaurants">here</a> for lunch options.</p>
+  </li>
+  <li>
+    <p><strong>Organizers will provide free coffee break?</strong></p>
```
Comment (gvwilson) [inline index.html:278]: _Will the organizers provide a free coffee break?_

File: index.html [hunk 10]
```diff
@@ -491,6 +530,21 @@ <h4 id="r-linux">Linux</h4>
 </div>
 -->
 
+<h2 id="restaurants">Restaurants</h2>
+
+<p>Since the workshop will happen in a holiday, here is a few suggestions of
```
Comment (gvwilson) [inline index.html:535]: here _are_ a few suggestions

**Pull request merged**